.travis.yml

```diff
@@ -36,10 +36,13 @@ jobs:
       before_deploy:
         - poetry config repositories.mdsol https://mdsol.jfrog.io/mdsol/api/pypi/pypi-local
         - poetry config http-basic.mdsol $ARTIFACTORY_USERNAME $ARTIFACTORY_PASSWORD # Stored as Travis CI Environment Vars
+        - poetry config pypi-token.pypi $POETRY_PYPI_TOKEN_PYPI  # this may be unnecessary
```

Comment:
looks like there is an issue with Environment variable `POETRY_PYPI_TOKEN_<NAME>` so I think this is necessary for now.
https://github.com/python-poetry/poetry/issues/2210